pkg/splunk/enterprise/names.go

```diff
@@ -250,3 +271,38 @@ func GetSplunkImage(specImage string) string {
 func GetPortName(port string, protocol string) string {
 	return fmt.Sprintf(portNameTemplateStr, protocol, port)
 }
+
+// GetProbeConfigMapName uses a template to name a Kubernetes ConfigMap for a Liveness/Readiness Probe.
+func GetProbeConfigMapName(identifier string) string {
```

Comment:
Can't we have this as a constant like other [places](https://github.com/splunk/splunk-operator/blob/7098178a23c4a51f81f20027ecf0be7b0821b0e1/pkg/splunk/enterprise/names.go#L31) where we can fill it in with a fmt.Sprintf?

Reply:
This is a getter for constant `probeConfigMapTemplateStr`